cmd/deploy/start.go

```diff
@@ -25,18 +27,33 @@ const (
 )
 
 type deployStartOptions struct {
-	deploymentFile string
-	application    string
-	context        map[string]string
+	deploymentFile    string
+	application       string
+	context           map[string]string
+	waitForCompletion bool
+}
+
+type deploymentCmdStatus struct {
+	deploymentID    string
+	executionResult chan string
 }
 
 type FormattableDeployStartResponse struct {
 	// The deployment's ID.
-	DeploymentId string `json:"deploymentId,omitempty" yaml:"deploymentId,omitempty"`
-	httpResponse *_nethttp.Response
-	err          error
+	DeploymentId    string `json:"deploymentId,omitempty" yaml:"deploymentId,omitempty"`
+	ExecutionStatus string `json:"status,omitempty" yaml:"status,omitempty"`
+	httpResponse    *_nethttp.Response
+	err             error
 }
 
+const (
+	DeployResultPipelineID = "PIPELINE_ID"
```

Comment:
Same here - this should be `DEPLOYMENT_ID`. I will check with product because this will eventually become really confusing for everyone.